index.js

```diff
@@ -20,7 +20,10 @@ function readAsJSON(fileName) {
 function yamlMerge(...from) {
   const files = from.map((path) => readAsJSON(path));
 
-  const outputJSON = _.merge({}, ...files);
+  const outputJSON = _.mergeWith({}, ...files, (objValue, srcValue, key, object, source, stack) => {
+    if (Array.isArray(objValue) && Array.isArray(srcValue)) return [...objValue, ...srcValue];
+    return undefined; // handle it just like with _.merge
```

Comment:
Just to make sure I understand, returning `undefined` means "do whatever `lodash` would normally do by default" and not "replace with `undefined`", right?

Reply:
Yes, exactly, that's according to their specification.

Reply:
This now breaks my use cases so I am sticking to 2.0. Would it have been better to provide a customizer fn as a prop for the node package for custom use cases such as these.